### PR TITLE
SONARIAC-1164 Evaluate simple Helm Charts and raise Kubernetes issues

### PR DESCRIPTION
**Pull request by jonas-wielage-sonarsource**

### CODE REVIEW

File: its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M6/values.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+escalation: true
+#           ^^^^   #TODO: SONARIAC-1132 Raise here the secondary location
```
Comment (mstachniuk) [inline its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M6/values.yaml:2]: It should be here SONARIAC-1131 Raise a secondary location from where the sensitive value comes from

**Review by mstachniuk — APPROVED** [review]
LGTM and please see my small comments

File: its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M1/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+apiVersion: v2
```
[the remark this answers is not in the document]
Comment (jonas-wielage-sonarsource) [reply its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M1/Chart.yaml:1]: I wanted to produce valid helm file structures, which helm itself is able to evaluate.
That's why I created the minimum structure that is able to be evaluated with `helm template .`

File: its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M2/templates/pod.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: example
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: {{ .Values.protocol | default "TCP" | quote }} # TODO SONARIAC-1184 HelmEvaluator evaluateTemplate fails for default and toYaml function
```
[the remark this answers is not in the document]
Comment (jonas-wielage-sonarsource) [reply its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M2/templates/pod.yaml:12]: Actually it's both files.
See this ticket I created: https://sonarsource.atlassian.net/browse/SONARIAC-1184

File: its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M6/values.yaml (hunk 1 above)
Comment (jonas-wielage-sonarsource) [reply its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M6/values.yaml:2]: Changed

File: its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M1/Chart.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mstachniuk) [reply its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M1/Chart.yaml:1]: `helm template -s templates/range.yaml . ` for the rescue ;-)

[the remark this answers is not in the document]
Comment (jonas-wielage-sonarsource) [reply its/ruling/src/test/resources/sources/kubernetes/helm/milestone/L1M1/Chart.yaml:1]: Mh didn't work for now on my side, I will look at it tomorrow

**Pull request merged**